src/pacer.js

```diff
@@ -84,10 +84,21 @@ let PACER = {
       return match[0].slice(1);
     }
   },
+  
   // Returns true if the URL is for docket query page.
   isDocketQueryUrl: function (url) {
-    // The part after the "?" is all digits.
-    return !!url.match(/\/(DktRpt|HistDocQry)\.pl\?\d+$/);
+    // This regex expression will match URLs that has a query string with only
+    // digits or has one or multiple parameters separated by the ampersand ("&").
+    //
+    // This function will return true for the following URLs:
+    // 
+    //  https://ecf.mied.uscourts.gov/cgi-bin/DktRpt.pl?365816
+    //  
+    //  https://ecf.moed.uscourts.gov/cgi-bin/DktRpt.pl?caseNumber=4:19-cr-00820-SRC-1
+    //
+    //  https://ecf.mad.uscourts.gov/cgi-bin/DktRpt.pl?caseNumber=1:17-cv-10577&caseId=0
+    //
+    return !!url.match(/\/(DktRpt|HistDocQry)\.pl\?(\d+|(&?[\w]+=[^&=\n]+)+)$/);
```

Comment:
I'm still wondering why we shouldn't just be more lenient here. Why not something as simple as:

```suggestion
    return !!url.match(/\/(DktRpt|HistDocQry)\.pl\?.*/);
```

Would that over-include something we're trying to exclude?

Reply:
The extension uses that method to identify the **Docket Query page** and the **Docket Report Page**. The URL for the **Docket Query page** is the one that has only digits (e.g. `/DktRpt.pl?115206`) and the **Docket Report page** could have a query string with one or more parameters if the user clicks the link available in appellate PACER ( e.g. `/DktRpt.pl?caseNumber=2:16-cv-01129-RFB-DJA` or `/DktRpt.pl?caseNumber=1:17-cv-10577&caseId=0` according to https://github.com/freelawproject/recap/issues/222). Here are the screenshots for these two pages:


**Docket Query page**  |  **Docket Report Page**
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/55959657/205361469-a29d343e-50b2-4778-a390-cf6d93f15695.png) |  ![image](https://user-images.githubusercontent.com/55959657/205361139-9143a4bf-737e-49f7-94df-212170dd300a.png)

There's another page with an URL ending in `DktRpt.pl` but has a different format in its query string, so it is excluded by this function, The **docket display page** has a URL  like this `/DktRpt.pl?591030040473392-L_1_0-1`. Here's the screenshot of that page:

![image](https://user-images.githubusercontent.com/55959657/205361749-bf2e32ac-4ec5-4a63-b492-5b8643d7e4e3.png)
 

Using a regex that matches any character after the start of the query string will return `true` for the **docket display page**.

Reply:
Got it, thank you. Let's just clarify this in the comment, then and we should be good.

Reply:
I'll suggest some language. :)